apps/desktop-wallet/package.json

```diff
@@ -4,22 +4,32 @@
   "version": "2.1.5",
   "author": "Alephium dev <dev@alephium.org>",
   "main": "public/electron.js",
+  "homepage": "./",
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:alephium/alephium-frontend.git"
+  },
   "scripts": {
     "build": "tsc && cross-env INLINE_RUNTIME_CHUNK=false VITE_VERSION=$npm_package_version vite build",
     "start:electron": "concurrently \"cross-env VITE_VERSION=$npm_package_version vite\" \"electron .\"",
-    "electron-pack": "bun run build && electron-builder",
-    "electron-pack-windows": "bun run build && electron-builder -w",
-    "electron-pack-macos": "bun run build && electron-builder --mac --universal",
-    "electron-pack-ubuntu": "bun run build && electron-builder -l",
-    "electron-pack-linux:arm64": "bun run build && electron-builder -l --arm64",
+    "build:electron": "electron-builder",
+    "build:electron:windows": "electron-builder -w",
+    "build:electron:macos": "electron-builder --mac --universal",
+    "build:electron:ubuntu": "electron-builder -l",
+    "build:electron:linux:arm64": "electron-builder -l --arm64",
+    "ci:build": "turbo run build",
+    "ci:build:electron:ubuntu": "turbo run build:electron:ubuntu",
+    "ci:build:electron:windows": "turbo run build:electron:windows",
+    "ci:build:electron:macos": "turbo run build:electron:macos",
```

Comment:
To ensure that the `compile` task of every dependency of this app will run before the `build` does, we execute the turbo `build*` commands. See the workspace-specific `turbo.json` below.

FYI, we can have many turbo configs, like so:

```
./turbo.json
./apps/desktop-wallet/turbo.json
./apps/explorer/turbo.json
```

Very useful feature.

See more: https://turbo.build/repo/docs/core-concepts/monorepos/configuring-workspaces